crates/turbopack/src/resolve.rs

```diff
@@ -115,6 +115,53 @@ async fn base_resolve_options(
     let mut plugins = opt.plugins.clone();
     plugins.push(UnsupportedSassResolvePluginVc::new(root).as_resolve_plugin());
 
+    let conditions = {
```

Comment:
Hoisted so that it can be shared for both "into" and "in" conditions. No changes.